pep-0012.rst

```diff
@@ -568,35 +568,43 @@ the numbers will always match.
 Images
 ------
 
-If your PEP contains a diagram, you may include it in the processed
-output using the "image" directive::
+If your PEP contains a diagram or other graphic, you may include it in the
+processed output using the ``image`` directive:
+
+.. code-block:: rst
 
     .. image:: diagram.png
 
-Any browser-friendly graphics format is possible: .png, .jpeg, .gif,
-.tiff, etc.
+Any browser-friendly graphics format is possible; PNG should be
+preferred for graphics, JPEG for photos and GIF for animations.
+Currently, SVG must be avoided due to compatibility issues with the
+PEP build system, but this may change in the near future.
```

Comment:
```suggestion
Currently, SVG must be avoided due to compatibility issues with the
PEP build system.
```

Don't include speculation. I might add a note that SVG should be preferred as the master copy, and PNG renders can be used -- it would be nice to seamlessly transition when technology allows.